org.qt_project.qtcreator.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: org.qt_project.qtcreator
+runtime: org.kde.Sdk
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: qtcreator
+# Qt Creator already provides desktop launchers and AppData metadata
+# but since we cannot use the original AppID, we have to rename the original files
+rename-desktop-file: org.qt-project.qtcreator.desktop
+rename-appdata-file: org.qt-project.qtcreator.appdata.xml
+rename-icon: QtProject-qtcreator
+finish-args:
+# Qt Creator crashes GNOME Shell 3.36 on Wayland
+# - --socket=wayland
+# - --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --share=network
+- --device=dri
+- --filesystem=host
+cleanup:
+- /include
+- /lib/arch
+- /lib/cmake
+- /lib/libQt6Core5Compat.prl
+- /lib/metatypes
+- /lib/qtcreator/objects-RelWithDebInfo
+- /lib/x86_64-linux-gnu
+- /mkspecs
+- /modules
+- /qml
+- /share/cmake
+- /share/pkgconfig
```

Comment:
IMO it's a bad practice to remove development resources from a development related app.

Reply:
This is not an SDK. The dependencies are for building the IDE.

What would be the purpose of bundling only yaml-cpp and the Qt5 compatibility module with an IDE? You can hardly build a useful program out of those.